src/player/index.ts

```diff
@@ -232,11 +302,42 @@ function getNextTour(): CodeTour | undefined {
   }
 }
 
-function getHeaderText(step: CodeTourStep): string {
+async function getFileContext(
+  line: number,
+  file: string,
+  uri: Uri
+): Promise<string> {
+  let label = `This step is on line ${line} in file ${file}`;
+  const document = await workspace.openTextDocument(uri);
```

Comment:
future cleanup: we are making this call to get the document multiple times, we could probably do this better.

Reply:
I saw that, I suppose we could do something like have a lazy wrapper and init once?